ci/package-mariner.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
This should be merged into `package.sh` instead of being a separate file.

Reply:
merged into package.sh